### ISSUE TITLE
key? vs keys.include?

### ISSUE THREAD

**Issue by AlexVPopov** [bbatsov/rubocop#3707]
Can one make the argument that given a hash `my_hash` it is a better style to write:
```ruby
my_hash.key? some_value
```
instead of 

```ruby
my_hash.keys.include? some_value
````
? If yes - I have a question towards the more experienced among you - how hard is implementing such a check? With some help from you, I would volunteer to submit such a PR.

--------

## Expected behavior

Describe here how you expected RuboCop to behave in this particular situation.

## Actual behavior

Describe here what actually happened.

## Steps to reproduce the problem

This is extremely important! Providing us with a reliable way to reproduce
a problem will expedite its solution.

## RuboCop version

Include the output of `rubocop -V`. Here's an example:

```
$ rubocop -V
0.16.0 (using Parser 2.1.2, running on ruby 2.0.0 x86_64-darwin12.4.0)
```

**Comment by bbatsov** [comment]
> If yes - I have a question towards the more experienced among you - how hard is implementing such a check? With some help from you, I would volunteer to submit such a PR.

It's definitely better and it's trivial to implement. But as with all similar checks - there's some potential for false positives. After all you never know whether `my_hash` is really a hash.
